packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -209,16 +244,25 @@ export class TreeViewConfiguration<TSchema extends ImplicitFieldSchema = Implici
 }
 
 /**
- * Detect cases documented in {@link ITreeConfigurationOptions.ensureUnambiguous}.
+ * Pretty print a set of types for use in error messages.
+ */
+function formatTypes(allowed: Iterable<TreeNodeSchema>): string {
+	// Use JSON.stringify to quote and escape identifiers.
+	// Don't just use a single array JSON.stringify since that omits spaces between items
+	return `[${Array.from(allowed, (s) => JSON.stringify(s.identifier)).join(", ")}]`;
```

Comment:
```suggestion
	return `[${Array.from(allowed, (s) => s.identifier).join(", ")}]`;
```

Reply:
See comment two lines up for why JSON.stringify is used here.

Reply:
I didn't realize that identifiers might have characters that we need to escape.